test/blackbox-tests/test-cases/builtin-support-override.t

```diff
@@ -0,0 +1,42 @@
+Create a library called `findlib.dynload`
+
+  $ mkdir findlib
+  $ cat > findlib/dune-project <<EOF
+  > (lang dune 3.7)
+  > (package (name findlib))
+  > EOF
+  $ cat > findlib/dune <<EOF
+  > (library
+  >   (name findlib_dynload)
+  >   (public_name findlib.dynload)
```

Comment:
I think it's because this library needs to provide a `Dynlink` and `Findlib` modules like the actual `findlib.dynload` does via its dependencies